### PR TITLE
User Submission Interface

### PR DESCRIPTION
**Pull request by JeffThorslund**
Lets user submit river info through text and an attached picture. 

**To test, run command**
`npm run lambda-serve` to start the pseudo back end, then 
`npm start` in another terminal to run the react app.

Use your own email where mine is to test a submission. It will go to your email instead of opening a GitHub Issue. 

**./function/sendMail.js**
image

### CODE REVIEW

**Comment by JeffThorslund** [comment]
Also note this wont run on a  Netlify. It needs certain specific endpoints added before it can be pushed to master.

**Comment by JeffThorslund** [comment]
> Also note this wont run on a Netlify. It needs certain specific endpoints added before it can be pushed to master.

Actually it looks like it is running fine...I just tested it on this deployment preview.

https://5eb709374ceb980007de5cf1--nifty-shirley-7636b6.netlify.app/

File: functions/sendMail.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  async function main() {
+    // create reusable transporter object using the default SMTP transport
+    let transporter = nodemailer.createTransport({
+      host: "smtp.zoho.com",
+      port: 465,
+      secure: true, // true for 465, false for other ports
+      auth: {
+        user: "wetexittest@zohomail.com",
+        pass: "pullout123",
```
Comment (oliviaperryman) [inline functions/sendMail.js:15]: If this is a real email address, then this is not secure to push to a public github repo.. could maybe look into storing this as an environment variable.

File: src/components/_standards.scss [hunk 2]
```diff
@@ -22,3 +22,5 @@ $RockSalt: "Rock Salt";
 $Alata: "Alata";
 $BalooTamma2: "Baloo Tamma 2";
 $Rubik: "Rubik";
+
+/*---Mixins---*/
```
Comment (oliviaperryman) [inline src/components/_standards.scss:26]: why is this a part of this PR

File: src/components/river/River.js [hunk 3]
```diff
@@ -16,6 +18,7 @@ class River extends Component {
       level: 0,
       mapBool: false,
       symbolBool: false,
+      knowledgeBool: true,
```
Comment (oliviaperryman) [inline src/components/river/River.js:21]: I think this should be false... the panel should not be automatically open

File: src/river-data/RiverList.js [hunk 4]
```diff
@@ -29,5 +29,5 @@ const RiverList = [
     data: null,
     global: null,
   },
-];
+]; //
```
Comment (oliviaperryman) [inline src/river-data/RiverList.js:32]: what is this for

File: src/river-data/my-river-template/data.js [hunk 5]
```diff
@@ -9,7 +9,6 @@ export const global = {
   riverInfo:
     "Add information about put in location, take out location. Any fees or private property complications? Specific season that the river is runnable?",
   putInCoords: "45.747890, -76.792082",
-
```
Comment (oliviaperryman) [inline src/river-data/my-river-template/data.js:12]: was this necessary

File: functions/sendMail.js (hunk 1 above)
Comment (JeffThorslund) [reply functions/sendMail.js:15]: Good idea!

File: src/components/_standards.scss (hunk 2 above)
Comment (JeffThorslund) [reply src/components/_standards.scss:26]: I'm L E A R N I N G

File: src/components/river/River.js (hunk 3 above)
Comment (JeffThorslund) [reply src/components/river/River.js:21]: Agree. I had it "true" while developing and forgot to set "false" before pushing.

**Pull request merged**